tests/test_install.py

```diff
@@ -30,8 +30,8 @@ class CommonTest(support.ResultTestCase):
 
     - contains a package "lotus-3-17.x86_64" (The package can be installed.)
     - contains a package "lotus-3-17.i686" (The package can be installed.)
-    - contains a package "trampoline-2.1-1.noarch" that contains "*/there" and
-      provides "splendid > 2.0" (The package can be installed.)
+    - contains a package "trampoline-2.1-1.noarch" that contains "*/there",
```

Comment:
just a nitpick: could you please change ```"*/there"``` to ```"/all/*/there"``` so that it matches the new ```test_install_filename_glob```. please?